app/guides/[section]/[slug]/guides.css

```diff
@@ -0,0 +1,113 @@
+/* HTML Styling */
+.markdownArea {
+  color: #ecf0f5;
+
+  h1,
+  h2,
+  h3 {
+    padding-top: 0.5em;
+    font-weight: bold;
+    color: #d3bc8e;
+  }
+
+  h1 {
+    font-size: 2em;
+    text-align: center;
+  }
+
+  h2 {
+    font-size: 1.5em;
+  }
+
+  h6 {
+    font-size: 0.5em;
+  }
```

Comment:
So the markdown does allow for h6 if you do `###### this is an h6` I set the font size really small because something somewhere is setting all header tags to the same size... not sure what it is. I couldn't find it.

Reply:
Ideally all of these styles should “@apply” classes from tailwind to keep things consistent across the app